### PR TITLE
Terminology formatting updates for the 4.4 release notes

### PR DESCRIPTION
**Pull request by jeana-redhat**
API terms updates and formatting fixes (removing soft wraps) for 4.4 release notes.

### CODE REVIEW

**Comment by jboxman** [comment]
@jeana-redhat, a brief description of the PR and a link to a preview is always appreciated, thanks!

**Comment by jboxman** [comment]
@jeana-redhat, so I hadn't considered this, but by mingling wraps with actual changes, there's no way to disambiguate what changed and what didn't.

I'm not sure it's practical to peer review the entire release notes, particularly for 4.4 which will be EOL in 3 months anyway. All I can really do is a best guess.

**Comment by jeana-redhat** [comment]
> @jeana-redhat, a brief description of the PR and a link to a preview is always appreciated, thanks!

Oops, forgot that the preview won't generate with a `.` in the PR title. @adellape want me to upload one?

File: release_notes/ocp-4-4-release-notes.adoc [hunk 1]
```diff
@@ -1009,623 +843,243 @@ of removed metrics is next.
...
-
-*RHCOS*
-
-* Network teaming is now supported in RHEL CoreOS. The teamd and
-NetworkManager-team RPMs have been added to RHEL CoreOS, enabling the setup and
-management of teamed network devices.
-(link:https://bugzilla.redhat.com/show_bug.cgi?id=1758162[*BZ#1758162*])
+* Previously, `openshift-controller-manager` metrics were not properly registered with the 1.16 Kubernetes Prometheus registry. This caused missing metrics for the {product-title} control plane. With this update the `openshift-controller-manager` metrics are now properly registered and the missing {product-title} control plane metrics have been restored. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1810304[*BZ#1810304*])
+
+* Previously, pull secrets were sometimes not deleted when their associated token was deleted. This caused pull secrets to remain associated with Kubernetes service accounts. With this update references to the owner of a pull secret and the associated token secret have been established. Now ,when a pull secret is deleted the associated token is deleted as well. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1806792[*BZ#1806792*])
```
Comment (adellape) [inline release_notes/ocp-4-4-release-notes.adoc:989]: s/Now ,when/Now, when/

File: release_notes/ocp-4-4-release-notes.adoc [hunk 2]
```diff
@@ -1009,623 +843,243 @@ of removed metrics is next.
...
+
+* Previously, the rate limit for creating pull secrets in the internal registry was low. This caused long wait times when a large number of namespaces were created in a short time period. With this update the rate limit for creating pull secrets in the internal registry has been increased. Pull secrets can now be created quickly even during heavy load. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1819849[*BZ#1819849*])
+
+*{op-system}*
+
+* Network teaming is now supported in {op-system-first}. The teamd and `NetworkManager-team` RPMs have been added to {op-system}, enabling the setup and management of teamed network devices. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1758162[*BZ#1758162*])
```
Comment (adellape) [inline release_notes/ocp-4-4-release-notes.adoc:995]: `teamd` as a package name?

File: release_notes/ocp-4-4-release-notes.adoc [hunk 3]
```diff
@@ -1009,623 +843,243 @@ of removed metrics is next.
...
-new project creation and users can use any of the examples in a FIPS
-environment.
-(link:https://bugzilla.redhat.com/show_bug.cgi?id=1774318[*BZ#1774318*])
+* When using the `--param-file` option with template processing commands like `oc new-app` or `oc process`, if the file was greater than 64K in size, it would not be fully read in. This caused the  the `oc`-based template processing with `--param-file` to fail. {product-title} now checks the size of the file specified by `--process-file` and augments the parameters used to read the entire file. The `oc`-based template processing with `--param-file` pointing to files greater than 64K now works. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1748061[*BZ#1748061*])
+
+* Previously, one of the new-app/new-build examples used for project creation would cause an error in a FIPS environment because the example was not FIPS-compliant. Now, only FIPS-compliant new-app/new-build examples are shown on new project creation and users can use any of the examples in a FIPS environment. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1774318[*BZ#1774318*])
```
Comment (adellape) [inline release_notes/ocp-4-4-release-notes.adoc:1017]: `new-app` and `new-build` are `oc` commands.

File: release_notes/ocp-4-4-release-notes.adoc [hunk 4]
```diff
@@ -1636,65 +1090,25 @@ references the correct API and works with the web console UI.
...
-easier for the user to access the Che workspace.
-(link:https://bugzilla.redhat.com/show_bug.cgi?id=1780338[*BZ#1780338*])
-
-* Creating a Knative service using the CLI while the Topology view was open
-caused a GUI error. Checks have been added to handle this workload without
-causing a GUI error.
-(link:https://bugzilla.redhat.com/show_bug.cgi?id=1781188[*BZ#1781188*])
+* Serverless resources like the `Service` YAML file are listed as `v1beta1` instead of `v1`. However, `v1beta1` is deprecated. With this bug fix, the `apiVersion` is updated to `v1`. (link:https://bugzilla.redhat.com/show_bug.cgiid=1796421[*BZ#1796421*])
+
+* A service binding request (SBR) in the topology is associated with *Revisions* in the *Topology* view. Therefore, no new revision will get the associated secrets. The SBR should go through the knative service; secrets would be injected and new revisions would be created. (link:https://bugzilla.redhat.com/show_bug.cgiid=1798288[*BZ#1798288*]
```
Comment (adellape) [inline release_notes/ocp-4-4-release-notes.adoc:1095]: I'm guessing a lot of these lowercase/no-markup instances of "knative" could probably be uppercased "Knative".

**Review by adellape — COMMENTED** [review]
NICE. Just a few tiny things.

File: release_notes/ocp-4-4-release-notes.adoc [hunk 5]
```diff
@@ -825,14 +661,12 @@ You must use the `oc create secret` command instead.
...
-The `oc build-logs` command was deprecated in {product-title} 3.11 and has been
-removed. You must use `oc logs` instead.
+The `oc build-logs` command was deprecated in {product-title} 3.11 and has been removed. You must use `oc logs` instead.
 
 [id="ocp-4-4-deprecated-kubernetes-metrics-removed"]
 ==== Deprecated upstream Kubernetes metrics have been removed
 
-All deprecated upstream Kubernetes metrics have been removed. The complete list
-of removed metrics is next.
+All deprecated upstream Kubernetes metrics have been removed. The complete list of removed metrics is next.
```
[the remark this answers is not in the document]
Comment (jeana-redhat) [reply release_notes/ocp-4-4-release-notes.adoc:669]: agree, gonna change to "follows"

File: release_notes/ocp-4-4-release-notes.adoc (hunk 2 above)
Comment (jeana-redhat) [reply release_notes/ocp-4-4-release-notes.adoc:995]: Good eye!

**Pull request merged**